py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2530,7 +2530,7 @@ def aten_ops_sort(
 @dynamo_tensorrt_converter(torch.ops.aten.trunc.default)
 @enforce_tensor_types(
     {
-        0: (TRTTensor,),
+        1: (TRTTensor,),
```

Comment:
It doesn't seem that this modification should apply to `trunc` here

Reply:
Oh good catch! That was accidentally involved by rebase.